explorer-api/src/utils.rs

```diff
@@ -0,0 +1,19 @@
+use cosmwasm_std::Uint128;
+use mixnet_contract_common::Coin;
+
+pub fn unym_to_nym(unym: Coin, fallback_amount: Option<u128>) -> Coin {
```

Comment:
if we **really** need it, I think we have this functionality elsewhere, but I'd argue its redundant

Reply:
If there is a utility function that exists already, i'd be glad to use it, but I didnt find one, so if you can give me that I'll appreciate.

> but I'd argue its redundant

not sure to understand why